sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -117,12 +122,23 @@ export default {
       },
     } as MenuModel);
 
+    const offerings = ref({
```

Comment:
I noticed that we are just following the existing code, but why those objects must be declared with `ref`?